ReimaginedLauncher/MainWindow.axaml.cs

```diff
@@ -108,14 +108,29 @@ private async Task CheckLatestModVersionAsync()
         var filesResponse = await _nexusModsHttpClient.GetModFilesAsync("diablo2resurrected", 503);
         if (filesResponse?.Files == null || filesResponse.Files.Count == 0)
             return;
-        // Assume the first file is the latest (sorted by upload date desc)
         var latestFile = filesResponse.Files.OrderByDescending(f => f.UploadedTimestamp).FirstOrDefault();
         if (latestFile == null)
             return;
         var latestVersion = latestFile.Version;
         if (!string.IsNullOrEmpty(_localModVersion) && !string.IsNullOrEmpty(latestVersion) && !_localModVersion.Equals(latestVersion, StringComparison.OrdinalIgnoreCase))
         {
-            Notifications.SendNotification("Update Available");
+            // Get the download link for the latest file
+            var downloadLinkResponse = await _nexusModsHttpClient.GenerateDownloadLink("diablo2resurrected", 503, latestFile.FileId);
+            string downloadUrl = downloadLinkResponse.ToString();
```

Comment:
This will need to get updated with the NexusModsDownloadLinkResponse.First().URI once the modal is created